07/main07.py

```diff
@@ -0,0 +1,166 @@
+                'stats      - shows you the number of entries stored\n'
+                'add        - adds a new entry\n'
+                'delete     - deletes an entry\n'
+                'list       - shows you all entries in the phonebook\n'
+                'show       - tells you details about entry\n'
+                '\n'
+                'Enter command: ')
+
+    inp_l = str(inp.lower())
+
+    # list ---------------------------
+    if inp_l == 'list':
+        os.system('cls')
+        print('\n')
+        for k, v in phone_dic.items():
+            print(k, v)
+        list_inp = input('\n'
+                        'Enter anything to start over: ')
+        if list_inp is not '':
+            continue
+        else:
+            continue
+    # list ---------------------------
+
+    # stats ---------------------------
+    elif inp_l == 'stats':
+        print(f'\n'
+              f'This phonebook has:\n'
+              f'{dic_count_items(phone_dic)} entries\n'
+              f'')
+    # stats ---------------------------
+
+    # show ---------------------------
+    elif inp_l == 'show':
+        os.system('cls')
+        show_entry = input('\n'
+                          'Type name of the entry to show: ')
+        if show_entry in phone_dic.keys():
+            print(f'\n'
+                  f'Entry "{show_entry} : {phone_dic.get(show_entry)}" is present...')
+            time.sleep(3)
+        else:
+            print(f'\n'
+                  f'Entry "{show_entry}" was not found...')
+            time.sleep(2)
+    # show ---------------------------
+
+    # delete ---------------------------
+    elif inp_l == 'delete':
+        os.system('cls')
+        show_entry = input('\n'
+                          'Type name of the entry to delete: ')
+        if show_entry in phone_dic.keys():
+            del phone_dic[show_entry]
+            print(f'\n'
+                  f'Entry "{show_entry}" with a value has been deleted...')
+            time.sleep(2)
+        else:
+            print(f'\n'
+                  f'Entry {show_entry} was not found...')
+            time.sleep(2)
+    # delete ---------------------------
+
+    # add ---------------------------
+    elif inp_l == 'add':
+        os.system('cls')
+        new_entry = input('\n'
+                          'Type name for the new entry: ')
+        if new_entry not in phone_dic:
+            new_value = input('\n'
+                              'Type name for the new value: ')
+            phone_dic[new_entry] = new_value
+            os.system('cls')
+            print('\n'
+                  'New entry with a value has been added...')
+            time.sleep(2)
+        else:
+            os.system('cls')
+            print(f'\n'
+                  f'There is the "{new_entry}" already. '
+                  f'You cannot overwrite it but you can delete it '
+                  f'and then create it with a new value.')
+            add_inp = input('\n'
```

Comment:
Here you have 2 possible conditions to continue your code, in other words, you always continue your code. In this case, maybe you don't need extra input from user. You can replace all this part (lines 125 - 130) with only one continue line

Reply:
I agree with you, but since after typing the existing entry user receives warning message on a separate page (sort of), I thought it would convenient for a user to have enough time to read that message and then go back to the main menu by typing anything or just pressing Enter.

Reply:
The same here: nothing will change if you replace `if else` with a simple `continue`

Reply:
It is fixed now.